plugins/@grouparoo/postgres/__tests__/table-import/import-property.ts

```diff
@@ -307,58 +307,28 @@ describe("postgres/table/recordProperty", () => {
             sourceMapping,
             aggregationMethod,
           },
-          [{ op, key: "purchase", match: "Apple" }]
-        );
-        expect(value).toEqual(2);
-      });
-      test("string is case sensitive", async () => {
-        const value = await getPropertyValue(
-          {
-            column,
-            sourceMapping,
-            aggregationMethod,
-          },
-          [{ op, key: "purchase", match: "apple" }]
-        );
-        expect(value).toEqual(0);
-      });
-      test("date", async () => {
-        const value = await getPropertyValue(
-          {
-            column,
-            sourceMapping,
-            aggregationMethod,
-          },
-          [{ op, key: "date", match: "2020-02-15" }]
+          [{ op, key: "purchase" }]
         );
-        expect(value).toEqual(1);
+        expect(value).toEqual(6);
       });
-      test("timestamp", async () => {
+    });
+
+    describe("does not exist", () => {
+      const op = "notExists";
+      test("string", async () => {
         const value = await getPropertyValue(
           {
             column,
             sourceMapping,
             aggregationMethod,
           },
-          [{ op, key: "stamp", match: "2020-02-15 12:13:14" }]
+          [{ op, key: "purchase" }]
         );
         expect(value).toEqual(1);
       });
-      test("float", async () => {
-        const value = await getPropertyValue(
-          {
-            column,
-            sourceMapping,
-            aggregationMethod,
-          },
-          [{ op, key: "amount", match: "1.54" }]
-        );
-        expect(value).toEqual(2);
```

Comment:
another lost test maybe

Reply:
The diff on this is more confusing than helpful for some reason. I added tests for "exists" and am only testing if against strings. These tests are still there for "equals" as they should be.